shared-ui/src/main/java/org/hzontal/shared_ui/bottomsheet/VaultSheetUtils.kt

```diff
@@ -335,10 +336,15 @@ object VaultSheetUtils {
                         action.importAndDelete()
                     }
                     //Move action
-                    createFolderActionTV.text = createFolderLabel
-                    createFolderActionTV.setOnClickListener {
-                        vaultManageFilesSheet.dismiss()
-                        action.createFolder()
+                    if (!isCreateFolderVisible) {
+                        createFolderActionTV.visibility = View.GONE
```

Comment:
we can replace this with `createFolderActionTV.isVisible = isCreateFolderVisible `

Reply:
Sure